test/integration/BLACKBOX_README.md

```diff
@@ -0,0 +1,398 @@
+# TEAL Blackbox Toolkit: Program Reporting and Testing via Dry Runs
+
+**NOTE: to get math formulas to render here using Chrome, add the [xhub extension](https://chrome.google.com/webstore/detail/xhub/anidddebgkllnnnnjfkmjcaallemhjee/related) and reload**
+
+## Blackbox Testing Howto
+
+### What is TEAL Blackbox Testing?
+
+TEAL Blackbox Testing lets you treat your TEAL program as a black box that for every received input produces an output and other observable effects. You can create reports that summarize those effects, and turn the _reports_ into _program invariant conjectures_ which you then check with _sequence assertions_.
```

Comment:
I personally find this paragraph a little daunting, and it makes me think "do I really need this complicated tool?" (especially the second sentence)

I think bullets on line 51-53 are saying a similar thing, but itemized in a way that feels friendlier/easier to digest - lets me easily see I could just use "make basic assertions and validate them using dry runs" if I want to, to start. That sounds useful to me already!

Reply:
👍